oeis.py

```diff
@@ -106,12 +106,30 @@ def A000290(start=0, limit=20):
 
 
 @oeis
-def A000079(start=0, limit=20):
-    "Powers of 2: a(n) = 2^n."
-    seq = []
-    for n in range(start, limit):
-        seq.append(2 ** n)
-    return seq
+def A001462(n=6):
+
+    dp = [0] * (n + 1)
+
+    dp[1] = 1
+
+    for i in range(2, n + 1):
+
+        dp[i] = 1 + dp[i - dp[dp[i - 1]]]
+
+    return dp
+
+
+@oeis
+def A000079(start=0, limit=20, plot=False):
+    sequence = []
+    for i in range(start, start + limit):
+        sequence.append(2 ** i)
+
+    if plot:
+        plt.plot(sequence)
+        plt.show()
```

Comment:
Beware, by "moving" this function you restored old code, the `if plot:` does no longer exists.